### ISSUE TITLE
How to recover from shared team folder InputOutput error with log 'last written by an unknown device'

### ISSUE THREAD

**Issue by bryanlandia** [keybase/client#10858]
/keybase/team/XX was last written by an unknown device claiming to belong to user XX.  The device has possibly been revoked by the user.  

my log id: 37fcf80e814d340902ec011c

**Comment by bryanlandia** [comment]
`keybase fs ls (KBFS path)` gives error like `last written by an unknown device claiming to belong to user XX. The device has possibly been revoked by the user.`  What can that user do to allow the rest of the team to access?  `keybase rekey`?

**Comment by OmarIthawi** [comment]
Same issue here (as in exactly the same issue, since I work with @bryanlandia).

When I ls the files:
```
$ ls /keybase/team/XX
ls: cannot open directory '/keybase/team/XX': Input/output error
```

I logged out and logged in again and I did `$ keybase db nuke` but none helped. The ls output stayed the same after the nuke.

**Comment by strib** [comment]
Hi @bryanlandia and @OmarIthawi.  Sorry for the inconvenience.  Right now we have an issue where if the last writer of your team folder (either via KBFS or git) then resets their Keybase account, it leaves things in a bad state.  We have two options for recovering this until we fix the underlying bug:

* Easiest: resetting your entire KBFS folder, losing whatever data is in the file system and in git, and starting fresh.
* Will take slightly longer, but happy to do it: Roll back just the most recent writes of the folder that user XX made.

If you don't have anything worth saving in KBFS, I'd slightly prefer the first option, but happy to do either, just let me know.

**Comment by bryanlandia** [comment]
@strib Thank you.  This is is good to know for now.  Somehow one of our team members was still able to access it, and we copied the contents into a newly created team.  When you fix the bug, I would be great if you would comment on this Issue so we see it.  👍

**Comment by strib** [comment]
I just pushed a fix for this to master: keybase/kbfs#1530.  It will be included in the next releases on all platforms, probably early next week.  Sorry for the delay and the inconvenience!
